cmd/producer/main.go

```diff
@@ -29,11 +29,11 @@ func main() {
 
 	// Create Kafka Producer
 	pChannels := kafka.CreateProducerChannels()
-	kafkaProducer, err := kafka.NewProducer(ctx, config.KafkaAddr, config.HelloCalledTopic, pChannels, &kafka.ProducerConfig{
+	kafkaProducer, err := kafka.NewProducer(ctx, config.KafkaAddr, config.PublishedContentTopic, pChannels, &kafka.ProducerConfig{
```

Comment:
this should be using something like `pConfig` which would be created beforehand and optionally includes the `SecurityConfig` for TLS